### PR TITLE
Update SetPedPopulationBudget.md

### PR DESCRIPTION
**Pull request by smoke1974**
Before you submit this PR, please make sure:

- You have read the contribution guidelines
- You include an example that validates your change
- Your English is grammatically correct

### CODE REVIEW

File: STREAMING/SetPedPopulationBudget.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ ns: STREAMING
 
 ```c
 // 0x8C95333CFC3340F3 0xD2D026CD
-void SET_PED_POPULATION_BUDGET(int p0);
+void SET_PED_POPULATION_BUDGET(int p5);
```
Comment (PsychoShock) [inline STREAMING/SetPedPopulationBudget.md:8]: Hello,

Is this an error? Cause this is wrong.

Thanks

**Comment by PsychoShock** [comment]
@4mmonium I think this was a mistake.

File: STREAMING/SetPedPopulationBudget.md (hunk 1 above)
Comment (4mmonium) [reply STREAMING/SetPedPopulationBudget.md:8]: This is the `budgetLevel` and it ranges from `0` to `4`.

Comment (PsychoShock) [reply STREAMING/SetPedPopulationBudget.md:8]: Isn't from `0` to `3`? Lowest is `0` for no peds and `3` is full budget.

Comment (4mmonium) [reply STREAMING/SetPedPopulationBudget.md:8]: > Isn't from `0` to `3`? Lowest is `0` for no peds and `3` is full budget.

You're correct, my bad. Will edit my reply.

**Pull request merged**